apps/sms/views/shared/js/activity_handler.js

```diff
@@ -318,8 +318,9 @@ var ActivityHandler = {
             data: { id, threadId }
           };
 
-          return titlePromise.then((title) => {
-            var notification = new Notification(title, options);
+          return titlePromise.then((title) =>
+            NotificationHelper.send(title, options)
```

Comment:
Oh, thanks for the reminder! We handled notification in `Utils.closeNotificationsForThread` so we should not let the helper to close the notification.